.github/workflows/generate-components.yaml

```diff
@@ -0,0 +1,40 @@
+name: Static TypeScript component update from main Octant repo
+on:
+  schedule: 
+  - cron: "0 0 * * *"
+
+jobs:
+  update-components:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.16.0'
+    - name: Install NodeJS
+      uses: actions/setup-node@v2
+      with: 
+        node-version: 14
```

Comment:
It's safe to bump to 16 here since LTS is coming in a month and we've moved upstream CI already